### PR TITLE
feat(landing-page): introduce landing page

### LINKED ISSUE #7: Landing page

### PR DESCRIPTION
**Pull request by chiptus**
fix #7

### CODE REVIEW

File: web/components/Landing/ContactForm.js [hunk 1]
```diff
@@ -1,22 +1,69 @@
...
-          type="text"
-          className="hover:bg-blue-100 w-full border-0 border-b border-blue-500 border-solid"
-        />
-        <button className="px-5 py-2 text-white bg-yellow-500 rounded-sm">
-          Join the mailing list
-        </button>
-      </div>
-    </form>
+              <div className="flex flex-col items-center justify-center mt-10 space-y-10">
+                <input
```
Comment (ameedjadallah) [inline web/components/Landing/ContactForm.js]: for the first time, I couldn't figure that this is an input field only when I clicked on the "join the mailing list", it looks like a divider, not an input field

File: web/components/Home/HowItWorks.js [hunk 2]
```diff
@@ -30,25 +30,19 @@ export default function HowItWorks({ flowCharts }) {
       </div>
       <div className="flex items-center space-x-24 select-none">
         {currentChart.steps.map((step, index, steps) => (
-          <>
-            <div
-              className="relative flex flex-col items-center"
-              key={step.title}
-            >
+          <Fragment key={step.title}>
```
Comment (ameedjadallah) [inline web/components/Home/HowItWorks.js:33]: I am thinking even if the title is unique, but do you think it's a good practice to use it as a key?
maybe the step number or the index

File: web/components/Home/Section.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ export default function Section({ children }) {
   return (
     <div className="bg-gradient-to-br from-blue-900 to-blue-500 relative w-full pb-20 overflow-x-hidden text-white">
       {children}
-      <div className="absolute bottom-0 flex">
+      <div className="-bottom-1 absolute flex">
```
Comment (ameedjadallah) [inline web/components/Home/Section.js:10]: I am not sure stating the class name with "-" is the better way, and at the same time I know it's represented a size with a minus, I've tried to search about it but couldn't found the best way to define sizes with minus values in tailwind

File: web/components/Home/Hero.js [hunk 4]
```diff
@@ -14,7 +14,11 @@ export default function Hero({ title, about }) {
     <Section>
       <div className="pt-30 sm:px-0 px-10 pb-40">
         <div className="top-5 left-5 absolute">
-          <Logo />
+          <Link href="/">
+            <a>
```
Comment (ameedjadallah) [inline web/components/Home/Hero.js:19]: Why adding "a" tag inside Link, I think Link is enough in React

Comment (chiptus) [reply web/components/Home/Hero.js:19]: this is how it's done https://nextjs.org/docs/routing/introduction#linking-between-pages

File: web/components/Home/Section.js (hunk 3 above)
Comment (chiptus) [reply web/components/Home/Section.js:10]: that's not a class I made, that's how tailwind does negative values

File: web/components/Home/HowItWorks.js (hunk 2 above)
Comment (chiptus) [reply web/components/Home/HowItWorks.js]: fixed

File: web/components/Landing/ContactForm.js (hunk 1 above)
Comment (chiptus) [reply web/components/Landing/ContactForm.js]: https://github.com/tezos-israel/wantez/issues/19

**Review by ameedjadallah — APPROVED** [review]
I've reviewed all my comments and all make sense, and as you mentioned we will work on the contact input field design.
Thanks.

**Pull request merged**